pkg/collector/upgrade/v0_111_0_test.go

```diff
@@ -34,9 +34,20 @@ func Test0_111_0Upgrade(t *testing.T) {
 	defaultCollectorWithConfig := defaultCollector.DeepCopy()
 
 	defaultCollectorWithConfig.Spec.Config.Service.Telemetry = &v1beta1.AnyConfig{
-		Object: map[string]interface{}{
-			"metrics": map[string]interface{}{
-				"address": "1.2.3.4:8888",
+		Object: map[string]any{
+			"metrics": map[string]any{
```

Comment:
is this still supposed to be happening? I thought only the upgrade step would be making this change?

Reply:
Sorry. I thought I reverted this. Since the method `ApplyDefaults` has change and this test will not pass otherwise, I will port the logic to the `upgrade0_111_0.go` file.